site/content/en/docs/Installation/_index.md

```diff
@@ -45,17 +43,17 @@ Each version of Agones supports a specific version of Kubernetes. When a new ver
 
 The following table lists recent Agones versions and their corresponding required Kubernetes versions:
 
-| Agones version | Kubernetes version |
-| -------------- | ------------------ |
-| 1.29           | 1.24               |
-| 1.28           | 1.23               |
-| 1.27           | 1.23               |
-| 1.26           | 1.23               |
-| 1.25           | 1.22               |
-| 1.24           | 1.22               |
-| 1.23           | 1.22               |
-| 1.22           | 1.21               |
-| 1.21           | 1.21               |
+| Agones version | Kubernetes version   |
+| -------------- | ------------------   |
+| 1.29           | {{% k8s-version %}}  |
```

Comment:
Nit: This just renders as `[1.23 1.24 1.25]`, do we want to do any better? Like commas or anything? I dunno, I'm not sure it's worth it.

Similarly do we want consistency through the rest of the table?

Reply:
Good call out. Did some trick to make it looks nicer.